catalog/aws.go

```diff
@@ -231,11 +217,15 @@ func (a *awsSyncer) fetchHealths(id string) (map[string]health, error) {
 	for paginator.HasMorePages() {
 		p, err := paginator.NextPage(context.TODO())
 
+		var notFound *awssdtypes.InstanceNotFound
+		if errors.As(err, &notFound) {
+			// Note (dans): I think this is the case when all the instances have an unknown health status for a service,
+			// which is fairly common because we don't sync the health status from Consul to AWS (yet).
+			a.log.Trace("instance not found", "service-id", id)
+			return result, nil
+		}
```

Comment:
This was a bug from my update of dependencies. We should be skipping over the health results when they are empty.